draft-ietf-httpbis-digest-headers.md

```diff
@@ -509,13 +509,13 @@ The SRI `integrity` attribute contains a cryptographic hash algorithm and digest
 value which is similar to `representation-data-digest` (see
 {{representation-digest}}). The major differences are in serialization format.
 
-The SRI digest value is calculated over the identity encoding of the resource,
-not the selected representation (as specified for `representation-data-digest`
-in this document). Section 3.4.5 of [SRI] describes the benefit of the identity
-approach - the SRI `integrity` attribute can contain multiple algorithm-value
-pairs where each applies to a different identity encoded payload. This allows
-for protection of distinct resources sharing a URL. However, this is a contrast
-to the design of representation digests, where multiple `Digest` field-values
+The SRI digest value is calculated over the representation data of a resource
+without any content-coding applied.
+Section 3.3.5 of [SRI] describes the benefit of not applying content-codings:
+the SRI `integrity` attribute can contain multiple algorithm-value
+pairs where each applies to a different representation data,
+thus protecting different representations of a given resource. However, this is different
```

Comment:
Fix 3: if it's the same url, then it's the same resource, which may have different representation => different representation data (eg. content-language, content-encoding, ...)